.github/workflows/release.yml

```diff
@@ -15,6 +15,8 @@ jobs:
 
         steps:
             - uses: actions/checkout@v3
+              with:
+                  node-version: "20"
 
```

Comment:
This doesn't seem to be necessary here. It's already covered in the next step (`Use Node.js`)

Reply:
At one stage during development, something failed (can't remember the details), and I needed to add that to fix the problem. I got rid of those 2 lines now, and recommitted, and all the tests pass. So I'm happy to leave them out.